packages/devextreme/testing/tests/DevExpress.ui.widgets.htmlEditor/htmlEditorParts/mentionModule.tests.js

```diff
@@ -441,8 +464,10 @@ QUnit.module('Mentions module', moduleConfig, () => {
             return;
         }
 
+        const totalItems = 60;
+        const pageSize = 50;
```

Comment:
Why does your change affect this test?

Reply:
This test remains with unchanged behavior. But locally it was red, so I did a little refactoring while debugging.